plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -124,33 +124,36 @@ const getPluginContent = async ({wiki, pluginName, reporter, createNode, createC
                 return createWikiNode('text/asciidoctor', url, body);
             }
         }
-        const promises = [
-            {filename: 'README.md', type: 'text/markdown'},
-            {filename: 'readme.md', type: 'text/markdown'},
-            {filename: 'README.adoc', type: 'text/asciidoctor'},
-            {filename: 'readme.adoc', type: 'text/asciidoctor'},
-        ].map(async type => {
-            try {
-                let url = wiki.url.replace('https://github.com/', 'https://raw.githubusercontent.com/');
-                if (!url.includes('/tree/')) {
-                    url = url.split('/').concat([defaultBranch || 'master']).join('/');
+        if (wiki.url.startsWith('https://github.com/') || wiki.url.startsWith('https://www.github.com/')) {
+            const promises = [
+                {filename: 'README.md', type: 'text/markdown'},
+                {filename: 'readme.md', type: 'text/markdown'},
+                {filename: 'README.adoc', type: 'text/asciidoctor'},
+                {filename: 'readme.adoc', type: 'text/asciidoctor'},
+            ].map(async type => {
+                try {
+                    let url = wiki.url.replace('https://github.com/', 'https://raw.githubusercontent.com/');
+                    if (!url.includes('/tree/')) {
```

Comment:
This `if` may need an `else`

Reply:
can you give me an example of why?

its passing on https://ci.jenkins.io/blue/rest/organizations/jenkins/pipelines/Infra/pipelines/plugin-site/branches/PR-961/runs/4/nodes/33/steps/39/log/?start=0

the only one that triggers "has no content that can be fetched from" that is github related that I can see is "error DotCi-Fig-template has no content that can be fetched from http://github.com/DotCi/DotCi-Fig-templat"

I nudged ulli on slack asking if he knew how to setup a warnings-ng report for these lines.

Reply:
btw, anything i saw with tree/ in it, already had the branch/ref
```
 ☎  ~  %   curl -qLs 'https://updates.jenkins.io/current/plugin-documentation-urls.json' | jq '.' | grep tree -B 1
  },
  "absint-astree": {
--
  "cloudify": {
    "url": "https://github.com/jenkinsci/cloudify-plugin/tree/cloudify-1.0.8"
--
  "ibm-application-security": {
    "url": "https://github.com/jenkinsci/appscan-plugin/tree/deprecatedPlugin"
--
  "kryptowire": {
    "url": "https://github.com/jenkinsci/kryptowire-plugin/tree/master/docs"
```

Reply:
I didn't test it locally, but I think the string `tree/` should be removed from the URL. The log you linked says it's downloading from https://raw.githubusercontent.com/jenkinsci/cloudify-plugin/tree/cloudify-1.0.8/README.md , should be just https://raw.githubusercontent.com/jenkinsci/cloudify-plugin/cloudify-1.0.8/README.md (failed fetch is currently not logged, AFAIU).

Reply:
oh good catch!

> failed fetch is currently not logged, AFAIU).

failed as in 404? I specifically disabled that because 3/4 of those URLs would be 404 and that's a lot of noise. I'll add that plugin to my specific tests and try again.

Reply:
maybe only log something if all 4 attempts end up as 404? (Non-blocking either way.)

Reply:
thats what I tried to do, I should abstract it into a function so I can test it more.

Reply:
fixed, but i havn't had time to fully locally test it (been busy helping infra stabilize), so gonna see what CI does for a full build